index.html

```diff
@@ -1,299 +1,318 @@
-                    </div>
-                    <div class="tags">
-                        <button class="htmlbutton" type="button">HTML </button>
-                        <button class="cssbutton" type="button">CSS </button>
-                        <button class="javascriptbutton" type="button">JavaScript </button>
-
-                    </div>
-                    <div class="action_button">
-                        <button class="button2" type="button">See Project</button>
-                    </div>
+      <nav class="card" id="two">
+        <div id="img2"></div>
+        <div class="left-bloks"  id="sub-left-block">
+          <div class="primary">
+            <h1>Multi-Post Stories</h1>
+            <div class="frame">
+              <p id="font">FACEBOOK</p>
+              <img src="./image/Counter.png" alt="counter" />
+              <h6>Back End Dev</h6>
+              <img src="./image/Counter.png" alt="counter" />
+              <h6>2015</h6>
+            </div>
+          </div>
+          <div class="textsecond">
+          <p class="text">
+            Experimental content creation feature that allows users existing 
+            story over  without spamming their friends.
+          </p>
+          <p class="sps"> </p>
+        </div>
+        <div class="thirdtext">
+          <ul>
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Ruby on rails</a></li>
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">JavaScript</a></li>
+          </ul>
+        </div>
+        <div class="fourthtext">
+          <div class="action">
+            <button type="submit" class="btn"><p>See project</p></button>
+          </div>
+        </div>
+      </nav>
 
-                </div>
+      <nav class="card" id="three">
+        <div id="img3"></div>
+        <div class="left-bloks" id="sub-left-block">
+          <div class="primary">
+            <h1>Facebook 360</h1>
+            <div class="frame">
+              <p id="font">FACEBOOK</p>
+              <img src="./image/Counter.png" alt="counter" />
+              <h6>Back End Dev</h6>
+              <img src="./image/Counter.png" alt="counter" />
+              <h6>2015</h6>
+            </div>
+          </div>
+          <div class="textsecond">
+          <p class="text">
+            Exploring the future of media in Facebook's place to 
+            discover and enjoy 360 photos and videos on Gear VR.
+          </p>
+          <p class="sps"> </p>
+        </div>
+        <div class="thirdtext">
+          <ul>
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Ruby on rails</a></li>
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">JavaScript</a></li>
+          </ul>
+        </div>    
+    <div class="fourthtext">        
+          <div class="action">
+            <button type="submit" class="btn" id="buto"><p>See project</p></button>
+          </div>
+        </div>
+      </nav>
```

Comment:
- [ ] You have made the cards too small for the desktop version. Pleas folllow the figma design closely.
![full](https://github.com/samuelkinyanjui/-finish_mobile_version_rev/assets/99656687/c622c29c-ad9e-437b-be37-1fcb7c857ff5)